crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -181,6 +185,46 @@ pub enum WalletInfo {
     GooglePay(GpayInfo),
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+#[serde(untagged)]
+pub enum BankRedirectInfo {
+    Ideal(IdealInfo),
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+pub struct IdealInfo {
+    pub issuer_id: Option<String>,
+}
+
+pub struct MultisafepayBankNames<'a>(&'a str);
+
+impl<'a> TryFrom<&BankNames> for MultisafepayBankNames<'a> {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(bank: &BankNames) -> Result<Self, Self::Error> {
+        Ok(match bank {
+            BankNames::AbnAmro => Self("0031"),
+            BankNames::AsnBank => Self("0761"),
+            BankNames::Bunq => Self("4371"),
+            BankNames::Ing => Self("0721"),
+            BankNames::Knab => Self("0801"),
+            BankNames::N26 => Self("9926"),
+            BankNames::NationaleNederlanden => Self("9927"),
+            BankNames::Rabobank => Self("0021"),
+            BankNames::Regiobank => Self("0771"),
+            BankNames::Revolut => Self("1099"),
+            BankNames::SnsBank => Self("0751"),
+            BankNames::TriodosBank => Self("0511"),
+            BankNames::VanLanschot => Self("0161"),
+            BankNames::Yoursafe => Self("0806"),
+            BankNames::Handelsbanken => Self("1235"),
+            _ => Err(errors::ConnectorError::NotSupported {
```

Comment:
Please don't use default here, rather mention all the enums. It will help us easily debug any issues when new enum is added.